build-docker.sh

```diff
@@ -37,8 +38,13 @@ case $key in
     echo "not implemented, just hardcode me :'("
     exit 1
     ;;
+    --python-version)
+    # Python version to install. e.g. 3.7.7
+    shift
+    PYTHON_VERSION=$1
+    ;;
```

Comment:
Can you just add a comment stating that changing python versions may require a different wheel

Reply:
Thank @ijrsvt for reviewing this.
Just addressed your comment.

Reply:
Perfect--looks good!